.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build and validate package
+
+on:
+  workflow_call:
+    inputs:
+      target_path:
+        description: "The path of the package or service to build"
+        required: true
+        type: string
+      run_test:
+        description: "Run lint and test"
+        required: true
+        type: boolean
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.8, 3.9, "3.10", "3.11" ]
```

Comment:
should consider the change we made in dbt, as Python 3.8 is end of life, Python 3.12 has been out for a while, and python 3.13 was released last month.